packages/gatsby/package.json

```diff
@@ -180,7 +180,7 @@
     "cross-env": "^7.0.3",
     "documentation": "^13.1.0",
     "enhanced-resolve": "^4.2.0",
-    "lmdb-store": "^1.5.0",
+    "lmdb-store": "~1.5.5",
```

Comment:
Why can't we use the latest beta yet? Does beta.3 also have bugs?

Reply:
Yes, our tests caught several more edge cases: https://github.com/DoctorEvidence/lmdb-store/issues/69#issuecomment-875480087